### ISSUE TITLE
Discount was Added to Order just because Gender was updated (Sammie) 424469

### ISSUE THREAD

**Issue by chris206** [206Tours/DB#1887]
Not good.

We DONT want discounts to be added automatically to any orders. If clients dont request them, or booked before the discount, we absolutely dont want to just cut their already agreed prices.

![image](https://cloud.githubusercontent.com/assets/22382144/22905438/38b91c2c-f20e-11e6-8361-1a7bb32cd9e1.png)

**Comment by bradatkins** [comment]
Your description does not match what Sammie said? 

You said the discount was added automatically, but same said she wants it to show up in the audit?

PS: I can't see the order #.

This does not look like the discount was added magically, but Sammie added it

**Comment by chris206** [comment]
I'm sorry, I dont know why I posted it that way BUT the way I described it is accurate

1. Sammie changed the gender on a passenger and hit update
2. The Valentines day discount appeared on the invoice where it wasnt before
3. Sammie went to check WHO added the discount and found no mention in audit

Deduction: the discount was added when she updated the booking

Solution: a. stop the database from auto-adding discounts
              b. add the addition of a discount as an event in the audit

![image](https://cloud.githubusercontent.com/assets/22382144/22977670/a2ae12ae-f35d-11e6-9e68-0ff6384a79b7.png)
